pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixInstanceDataManager.java

```diff
@@ -151,6 +151,15 @@ public void removeSegment(@Nonnull String tableNameWithType, @Nonnull String seg
     }
   }
 
+  @Override
+  public void deleteSegment(@Nonnull String tableNameWithType, @Nonnull String segmentName) {
+    LOGGER.info("Tracking deleted segment: {} from table: {}", segmentName, tableNameWithType);
```

Comment:
I think we have other logs for dropping a segment, do we need this one as well? If we really do, then it should go inside the 'if' statement

Reply:
Removed log.